wagtail/admin/templates/wagtailadmin/collections/_privacy_switch.html

```diff
@@ -7,7 +7,14 @@
     {% dialog icon_name='view' id='set-privacy' data_url=privacy_url title=_("Change privacy") message_status='info' message_heading=_("Privacy settings determine who is able to view documents in this collection.") %}
     {% enddialog %}
 
-    <div class="privacy-indicator {% if is_public %}public{% else %}private{% endif %}">
+    <div
```

Comment:
Maybe let's keep this convention from the other files and add in this file also.

```suggestion
    <div {% comment %} PUBLIC {% endcomment %}
```

Reply:
I think, it is not good to add just public comment `{% comment %} PUBLIC {% endcomment %}` here because the div can public or private itself instead we can place comment like `{% comment %} PUBLIC or PRIVATE {% endcomment %}`

Reply:
Ok. Happy to not do this. Please remove it from the other template changes also.